internal/eval/eval_test.go

```diff
@@ -10,17 +10,14 @@ import (
 	"testing"
 	"time"
 
-	"github.com/dicedb/dice/internal/server/utils"
-
-	"github.com/bytedance/sonic"
-	"github.com/dicedb/dice/internal/server/utils"
```

Comment:
How did this pass the CI? At line 13 and 16 the same package is imported.

Reply:
Not sure how it passed, was merged as part of PR. Lint doesn't show any error. Will raise a PR to fix.
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/68124a8c-8192-4265-8de0-12e97224e33b">

Reply:
Some buffering issue? Very strange. Seeing for the first time.